src/app/(not-authenticated)/sign_in/sign_in.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import BMLink from '../../../components/BMLink';
+import logo from '../../../../public/Title.png';
+import Input from '../../../components/Input';
+import Button from '../../../components/Button';
+import Image from 'next/image';
+
+export default function SignIn() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const router = useRouter();
+
+  const myButtonFunction = () => {
+    3;
+  };
```

Comment:
These all are here just so it works, the real functionality will be added in the next PR with the API